microcosm_logging/factories.py

```diff
@@ -189,15 +189,25 @@ def make_loggly_handler(graph, formatter):
     """
     base_url = graph.config.logging.loggly.base_url
     metric_service_name = environ.get("METRICS_NAME", graph.metadata.name)
-    loggly_url = "{}/inputs/{}/tag/{}".format(
-        base_url,
-        graph.config.logging.loggly.token,
-        ",".join([
-            graph.metadata.name,
-            graph.config.logging.loggly.environment,
-            metric_service_name
-        ]),
-    )
+    if metric_service_name != graph.metadata.name:
+        loggly_url = "{}/inputs/{}/tag/{}".format(
+            base_url,
+            graph.config.logging.loggly.token,
+            ",".join([
+                graph.metadata.name,
+                graph.config.logging.loggly.environment,
+                metric_service_name,
+            ]),
+        )
+    else:
+        loggly_url = "{}/inputs/{}/tag/{}".format(
+            base_url,
+            graph.config.logging.loggly.token,
+            ",".join([
+                graph.metadata.name,
+                graph.config.logging.loggly.environment,
+            ]),
+        )
```

Comment:
I don't think you need to create a branch like this, especially given that 99 percent of both blocks are the same.

You have a couple of options:
* extract out the tags into their own list, and then append the metric name iff it's not already in there.
* put all the tags into a list, but dedupe by putting it into something like a `set`

Reply:
Makes sense. Addressed it in the latest commit.